python/src/uagents/protocol.py

```diff
@@ -315,7 +315,7 @@ def manifest(self) -> Dict[str, Any]:
 
         for schema_digest, model in all_models.items():
             manifest["models"].append(
-                {"digest": schema_digest, "schema": model.model_json_schema()}
+                {"digest": schema_digest, "schema": model.schema()}
```

Comment:
why the change here? didn't we add model_json_schema so we don't have to change?

Reply:
Apparently there was a missing positional argument meaning I forgot to add the class method decorator. It should work now without the change.